v21.1/build-a-nodejs-app-with-cockroachdb-sequelize.md

```diff
@@ -55,16 +55,8 @@ Open `app.js`, and edit the connection configuration parameters:
 
 <section class="filter-content" markdown="1" data-scope="cockroachcloud">
 
-- At the top of the file, uncomment the `const fs = require('fs');` line.
```

Comment:
Don't they still need to specify the root cert to authenticate?

Reply:
No, I don't think so

Reply:
Okay - it will run, but the connection will be insecure as long as we have `rejectUnauthorized: false`, right? 

With `rejectUnauthorized: false` (the default), you'll get the following error: 
```
SequelizeConnectionError: server requires encryption
error: server requires encryption
```

I think one of the main pain points in the cloud connection story is accessing the root cert from a local directory. If we want to update our recommendations for simple "quick-start" connections to just not use root certs at all, then we will probably need to update a lot more docs pages.

@ianjevans @gemma-shay

Reply:
This config works:

~~~
var sequelize = new Sequelize({
  dialect: "postgres",
  username: "ian",
  password: "xxxxxxx",
  host: "free-tier.gcp-us-central1.cockroachlabs.cloud",
  port: 26257,
  database: "usable-weasel-4036.bank",
  dialectOptions: {
    ssl: true,
    native: true
  },
  logging: false,
});
~~~

Tried it after reading one of the answers to [this SO question](https://stackoverflow.com/questions/61153275/sequelizeconnectionerror-the-server-does-not-support-ssl-connections).

I tried a similar config using the regular connection string and the above `dialectOptions`, but I got an `SequelizeConnectionError: codeParamsRoutingFailed: missing cluster name in connection string` error, so Sequelize wasn't respecting the `options` query parameter.

Reply:
All right, I am reverting all cert-related stuff from this PR. Even were it a good idea, it should be a separate PR.